setup.py

```diff
@@ -167,6 +167,7 @@
             'hep_approval = inspirehep.modules.workflows.actions.hep_approval:HEPApproval',
             'merge_approval = inspirehep.modules.workflows.actions.merge_approval:MergeApproval',
             'match_approval = inspirehep.modules.workflows.actions.match_approval:MatchApproval',
+            'auto_non_core_record = inspirehep.modules.workflows.workflows:CoreSelection'
```

Comment:
are you sure this works? based on the other examples, it looks like the entry point should be a class with a `resolve` static method that deals with the payload, for example https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/workflows/actions/hep_approval.py#L33. I don't see anything like that in the class you reference (which is a workflow).

Reply:
Now for sure it's not working because this action is not registered at all, so I think that's the issue there https://sentry.inspirehep.net/inspire-prod/next/issues/36047/?query=is%3Aunresolved

Reply:
I know, but the thing you're registering doesn't have the right shape, it's a workflow class, not an action class (which has a `resolve` static method). Look at the example I linked.